packages/components/src/Description/Description.tsx

```diff
@@ -27,17 +27,25 @@ export type DescriptionProps = {
    * Определяет разделитель между Name Value
    */
   separator?: string;
+
+  /**
+   * Определяет тип HTML-элемента Description
```

Comment:
```suggestion
   * Определяет тип корневого HTML-элемента
```

Думаю стоит написать так
Так же можно указать пометку: Div стоит использовать при построении списков и наличии внешнего контейнера из `dl`